src/store/polls/types.ts

```diff
@@ -21,14 +32,27 @@ export interface PollsStateInterface {
   displayModal: boolean;
 }
 
+export interface PutPollInterface {
+  header: string;
+  description: string;
+  creationDate: string;
+  completionDate: string;
+  status: string;
+  polledHouses: Array<HouseModel>;
+}
+
 export type Mutations<S = PollsStateInterface> = {
   [PollsMutationEnum.SET_COOPERATION_POLLS](state: S, payload: Array<PollModel>): void;
   [PollsMutationEnum.SET_SELECTED_POLL](state: S, payload: number): void;
+  [PollsMutationEnum.DELETE_POLL](state: S, payload: any): void;
+  [PollsMutationEnum.UPDATE_POLL](state: S, payload: any): void;
```

Comment:
you forgot to add GET_POll_BY_ID

Reply:
GET_POll_BY_ID wasn't added by me, it's another developer's responsibility. I'll tell him